Jenkinsfile

```diff
@@ -23,9 +23,26 @@ node {
             sh 'git submodule update --init'
 
         stage 'StartVM'
+            // Calculate available amount of RAM
```

Comment:
Please change the stage variables to use block bodies{} as the alternative is deprecated